### PR TITLE
Nextjs-Chat-Application

### PR DESCRIPTION
**Pull request by divofred**

### CODE REVIEW

File: tutorials/next-chat-application/components/ActiveUsers.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+				Active Users
+			</h5>
+			{activeUsers.activeUsers
+				? activeUsers.activeUsers.map(item => (
+						<MDBCard key={Math.random()}>
+							<MDBCardBody>
+								<MDBTypography listUnStyled className="mb-0">
+									<li className="p-2">
+										<a
+											href="#!"
```
Comment (endymion1818) [inline tutorials/next-chat-application/components/ActiveUsers.js:17]: Should there be a page for the active user? If not you can remove this `a` tag

File: tutorials/next-chat-application/pages/api/hello.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (endymion1818) [inline tutorials/next-chat-application/pages/api/hello.js:1]: Can you remove this boilerplate code please

File: tutorials/next-chat-application/pages/api/socket.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+			//Adding the graphql query
+			query: `
+				{
+  		 listChats(sort: createdOn_ASC, limit: 20) {
+    		data {
+      		chats
+      		time
+      		user
+    		}
+  }
```
Comment (endymion1818) [inline tutorials/next-chat-application/pages/api/socket.js:21]: Please correct this formatting

**Review by endymion1818 — COMMENTED** [review]
Hey @divofred, thanks for your code submission! Just a few fairly minor things to sort out ...

File: tutorials/next-chat-application/components/ActiveUsers.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { MDBCol, MDBCard, MDBCardBody, MDBTypography } from 'mdb-react-ui-kit';
+
+const ActiveUsers = props => {
+	const { activeUsers } = props;
+	return (
+		<MDBCol md="6" lg="5" xl="4" className="mb-4 mb-md-0">
```
[the remark this answers is not in the document]
Comment (divofred) [reply tutorials/next-chat-application/components/ActiveUsers.js:6]: Done, sorry about that.

File: tutorials/next-chat-application/components/ActiveUsers.js (hunk 1 above)
Comment (divofred) [reply tutorials/next-chat-application/components/ActiveUsers.js]: I have modified it.

File: tutorials/next-chat-application/pages/api/hello.js (hunk 2 above)
Comment (divofred) [reply tutorials/next-chat-application/pages/api/hello.js]: Done.

File: tutorials/next-chat-application/pages/api/socket.js (hunk 3 above)
Comment (divofred) [reply tutorials/next-chat-application/pages/api/socket.js]: Done.

**Comment by divofred** [comment]
> Hey @divofred, thanks for your code submission! Just a few fairly minor things to sort out ...

Hi Ben, I've reviewed all the changes you requested for.

File: tutorials/next-chat-application/components/ActiveUsers.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (endymion1818) [reply tutorials/next-chat-application/components/ActiveUsers.js:6]: It doesn't look any different @divofred amd I missing something?

[the remark this answers is not in the document]
Comment (divofred) [reply tutorials/next-chat-application/components/ActiveUsers.js:6]: The problem is coming from my tab indent size. I changed it to 2 tab space but it reverts back to 8.

**Pull request closed without merging**